ethcore/blockchain/src/blockchain.rs

```diff
@@ -869,6 +854,14 @@ impl BlockChain {
 		}
 	}
 
+	/// clears all caches for testing purposes
+	pub fn clear_cache(&self) {
```

Comment:
if it's only used in tests, then put it behind `#[cfg(test)]` attribute

Reply:
Looks like it's not possible, this attribute doesn't work across crates (blockchain is a separate crate).